app/services/dialog_import_service.rb

```diff
@@ -113,6 +119,16 @@ def import_from_dialogs(dialogs)
           "resource_actions" => build_resource_actions(dialog)
         )
       )
+      associations.each do |a|
+        a.values.each do |g|
+          g.each do |v|
+            unless new_or_existing_dialog.dialog_fields.select { |f| f.name == v }.empty?
+              DialogFieldAssociation.create(:trigger_id => new_or_existing_dialog.dialog_fields.select { |f| f.name.include?(a.keys.first) }.first.id,
```

Comment:
You should be able to use `.find` here instead of `.select` since we're looking for the first element that matches.